R/completion.R

```diff
@@ -120,8 +120,12 @@ getCompletionList <- function(var, accessor, envs) {
     for (env in envs) {
       if (exists(var, env, inherits = FALSE)) {
         if (isPromise(var, env)) {
-          promise <- getPromiseVar(var, env)
-          obj <- eval(promise$promiseExpr, promise$promiseEnv)
+          if (getOption('vsc.previewPromises')){
```

Comment:
For the reason stated at https://github.com/ManuelHentschel/vscDebugger/pull/30/files#r434583833, we should supply a default value here.

If the default value is used in multiple places, like this one, we could put a variable (e.g. `vsc.previewPromises.default`) in `global.R` and use the variable here as the default value.